docs/units/format.rst

```diff
@@ -52,8 +52,16 @@ attributes within format strings::
     '10.500 in km'
 
 This might not work well with LaTeX strings, in which case it would be better
-to use the `Quantity.to_string() <astropy.units.Quantity.to_string()>`
-method::
+to use the `Quantity.to_string() <astropy.units.Quantity.to_string()>` method.
+Just like how `table.Column() <astropy.table.Column()>` takes a format specifier
+or callable for formatting, you can also optionally specify a format via the
+`formatter` parameter with either type or additionally with a dictionary, the
+added benefit being a more flexible or tighter LaTeX output. It relies on
+:func:`numpy.array2string()` for directly handling the :param:`formatter`, which will
```

Comment:
Hmm, readthedocs complains about `:param:` here and on l.63:
```
/home/docs/checkouts/readthedocs.org/user_builds/astropy/checkouts/16087/docs/units/format.rst:54: ERROR: Unknown interpreted text role "param".
/home/docs/checkouts/readthedocs.org/user_builds/astropy/checkouts/16087/docs/units/format.rst:54: ERROR: Unknown interpreted text role "param".
```
Simplest I think is just remove that and use double backquotes (also on l.58)